packages/valory/skills/liquidity_trader_abci/rounds.py

```diff
@@ -122,8 +120,6 @@ def participant_to_actions_round(self) -> DeserializedCollection:
     def actions(self) -> Optional[List[Dict[str, Any]]]:
         """Get the actions"""
         serialized = self.db.get("actions", "[]")
-        if serialized is None:
-            serialized = "[]"
```

Comment:
Why removed? `json.loads(None)` would fail. Same above.

Reply:
resolved in commit - ab03c744511b0180e22d51fe01c3531d68488890